tools/cldr-code/src/main/java/org/unicode/cldr/util/DtdData.java

```diff
@@ -160,6 +160,7 @@ public static class Attribute implements Named {
         private final Set<String> commentsPre;
         private Set<String> commentsPost;
         private boolean isDeprecatedAttribute;
+        private boolean isUEscapedAttribute = false;
```

Comment:
I think this needs to be isUEscap**able**Attribute instead, unless I misunderstand. That is, you are not *requiring* \\u escapes to be used in the field.

Reply:
If it says "escapable", as a consumer you don't know whether to unescape it or not. If it is "escaped", then it is clear you need to unescape it and that works regardless there is any \\u sequences in the string or not.

Reply:
I think the api should be revised to match the annotation we landed on, that is "ALLOWS_UESC".

Reply:
Perhaps attributeAllowsUescape?

Reply:
That works (and in function names etc)